Nadim.CinemaReservationSystem.Web/ClientApp/src/components/Cinema/FormCinemaRoomInfo.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import { Button } from 'react-bootstrap';
+
+export default class FormCinemaRoomInfo extends Component{
+    displayName = FormCinemaRoomInfo.displayName;
+
+    constructor(props){
+        super(props);
+        this.state = {
+            rows: '',
+            columns: '',
+            name: '',
+        }
+        this.handleSubmitClick = this.handleSubmitClick.bind(this);
+        this.handleCancelClick = this.handleCancelClick.bind(this);
+        this.handleRowsChange = this.handleRowsChange.bind(this);
+        this.handleColumnsChange = this.handleColumnsChange.bind(this);
+        this.handleNameChange = this.handleNameChange.bind(this);
+    }
+
+    handleSubmitClick(){
+        this.props.callBackReceiveCinemaRoomInfo({
+            rows : this.state.rows,
+            columns: this.state.columns,
+            name: this.state.name,
+        })
+    }
+
+    handleCancelClick(){
+        this.props.callBackCancel();
+    }
+
+    validateIntNumber(number){
+        let result = /^\d+$/;
+        return result.test(String(number));
+    }
+
+    handleRowsChange(event){
+        this.setState({
+            rows: event.target.value, 
+        });
+    }
+    
+    handleColumnsChange(event){
+        this.setState({
+            columns: event.target.value, 
+        });
+    }
+
+    handleNameChange(event){
+        this.setState({
+            name: event.target.value, 
+        });
+    }
+
+    render(){
+        return(
+            <div>
+                <h3>
```

Comment:
Here and everywhere - use semantic HTML markup:
- use "fieldset" for grouping form fields;
- use "label" for field labels;
- don't use "br" instead of margins/paddings;
- don't use "strong" instead of font weight;
etc